packages/block-editor/src/components/block-mover/index.js

```diff
@@ -44,24 +44,60 @@ export class BlockMover extends Component {
 	}
 
 	render() {
-		const { onMoveUp, onMoveDown, isFirst, isLast, isDraggable, onDragStart, onDragEnd, clientIds, blockElementId, blockType, firstIndex, isLocked, instanceId, isHidden, rootClientId } = this.props;
+		const { onMoveUp, onMoveDown, orientation, isRTL, isFirst, isLast, isDraggable, onDragStart, onDragEnd, clientIds, blockElementId, blockType, firstIndex, isLocked, instanceId, isHidden, rootClientId } = this.props;
 		const { isFocused } = this.state;
 		const blocksCount = castArray( clientIds ).length;
 		if ( isLocked || ( isFirst && isLast && ! rootClientId ) ) {
 			return null;
 		}
 
+		const getArrowIcon = ( hdir ) => {
```

Comment:
I think something like `moveDirection` which accepts either `'up'` or `'down'` would be a clearer argument here. That way the argument will match which direction the _block_ moves which is a constant instead of which direction the arrow faces which can change depending on `orientation` and `isRTL`.

Reply:
yes, horizontal is a modification of the normal vertical movement.